requirements/local-docs-build.txt

```diff
@@ -0,0 +1,9 @@
+mkdocs==0.14.0
```

Comment:
We should match all the versions that we are installing here with the ones that we are installing by default at 

https://github.com/rtfd/readthedocs.org/blob/8f2d59cdcdee0050feece79465d1a6c5c903ecf5/readthedocs/doc_builder/python_environments.py#L195-L228